packages/cli/src/formatters/junit.ts

```diff
@@ -46,7 +46,9 @@ export const junit: Formatter = (results, { failSeverity }) => {
       output += `<testsuite package="org.spectral" time="0" tests="${filteredValidationResults.length}" errors="0" failures="${filteredValidationResults.length}" name="${source}">\n`;
 
       for (const result of filteredValidationResults) {
-        output += `<testcase time="0" name="org.spectral.${result.code ?? 'unknown'}" classname="${classname}">`;
+        output += `<testcase time="0" name="org.spectral.${result.code ?? 'unknown'}${
+          result.path.length > 0 ? `(${xmlEscape(result.path.toString())})` : ''
```

Comment:
if you want `path` to be more readable, you could use `printPath` from `@stoplight/spectral-runtime`
[Examples](https://github.com/stoplightio/spectral/blob/develop/packages/runtime/src/utils/__tests__/printPath.test.ts)

Reply:
Good hint, updated!